playground/AspireEventHub/EventHubsApi/Program.cs

```diff
@@ -0,0 +1,28 @@
+using Azure.Identity;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Producer;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.AddAzureEventHubProducerClient("eventhubns", settings =>
+{
+    settings.EventHubName = "hub";
+    settings.Credential = new AzureCliCredential();
```

Comment:
```suggestion
```

This should stay `null` and just use `DefaultAzureCredential` when it isn't specified.